web/src/settings_components.ts

```diff
@@ -1099,6 +1102,20 @@ export function populate_data_for_stream_settings_request(
                     continue;
                 }
 
+                const stream_group_settings = new Set(["can_remove_subscribers_group"]);
+                if (stream_group_settings.has(property_name)) {
+                    const old_value = get_stream_settings_property_value(
+                        // eslint-disable-next-line @typescript-eslint/consistent-type-assertions
```

Comment:
I guess the fix for this probably involves some logic above that would type-narrow the value.